app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreen.kt

```diff
@@ -44,6 +45,12 @@ fun HomeScreen(
         }
     }
 
+    LaunchedEffect(Unit) {
+        viewModel.output.navigateToDetail.collect { id ->
+            navController.navigate("${CryptoScreen.COIN_INFORMATION.name}/$id")
+        }
+    }
```

Comment:
Could we move this to the `LaunchedEffect` block above?

Reply:
We have 2 LaunchedEffects here: error and `navigateToDetail`, I think that's fine 🤔